src/DiscordMessage.php

```diff
@@ -12,21 +12,31 @@ class DiscordMessage
     public $body;
 
     /**
-     * @param string $body
+     * The embedded object attached to the message.
+     *
+     * @var array
+     */
+    public $embed;
+
+    /**
+     * @param string     $body
+     * @param array|null $embed
      *
      * @return static
      */
-    public static function create($body = '')
+    public static function create($body = '', $embed = [])
```

Comment:
@messerli90 not sure if i'm missing something obvious but the $embed param here is unused? I had to construct an object to pass through an embed, static create() appears to do nothing.